README.md

```diff
@@ -896,4 +897,4 @@ make -j$( sysctl -in machdep.cpu.core_count )
 make install
 ```
 
-Your environment is set up. Now you can <a href="#runanode">build EOS and run a node</a>.
\ No newline at end of file
+Your environment is set up. Now you can <a href="#runanode">build EOS and run a node</a>.
```

Comment:
What's different here?  Did the line ending change and Github doesn't show it in the diff?

Reply:
yes, trailing white space removed